src/hooks/usePopup.ts

```diff
@@ -0,0 +1,29 @@
+import { useRouter } from 'next/router';
+import { useEffect } from 'react';
+import { useRecoilValue, useSetRecoilState } from 'recoil';
+
+import { popupAtom } from '@/store/components';
+
+export const usePopup = () => {
+  const router = useRouter();
+
+  const popupValue = useRecoilValue(popupAtom);
+  const setPopup = useSetRecoilState<PopupProps | null>(popupAtom);
```

Comment:
```suggestion
  const [popupValue, setPopupValue] = useRecoilState<PopupProps | null>(popupAtom);
```
위와 같이 한 줄로 표현 가능할 거 같아요!